requirements.txt

```diff
@@ -1 +1 @@
-numpy>=1.5
```

Comment:
Haha, wow, it's crazy looking back on that first commit. I was just wondering if this is something that deserves a changelog entry (saying we no longer support 1.5). I think I'm fine without it, since  `setup.py` has listed 1.6 as the requirement for a while now (maybe forever).